Assets/Scripts/SensitivityController.cs

```diff
@@ -0,0 +1,53 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class SensitivityController : MonoBehaviour
+{
+    [Header("UI Elements")]
+    [SerializeField] private Slider movementSlider;
+    [SerializeField] private Slider rotationSlider;
+
+    [Header("Sensitivity Values")]
+    public float movementSensitivity = 1.0f;
+    public float rotationSensitivity = 300f;
```

Comment:
publicにする必要がなければprivateにしておく。

```suggestion
    private float _movementSensitivity = 1.0f;
    private float _rotationSensitivity = 300f;
```